src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -669,6 +669,7 @@ final case class TenantApiServiceImpl(
     agreements <- agreementManagementService.getAgreements(producerId, consumerId, agreementStates)
     eServices  <- Future.traverse(agreements.map(_.eserviceId))(catalogManagementService.getEServiceById)
     attributeIds = eServices
+      .flatMap(_.descriptors.lastOption.toList)
```

Comment:
This is used to check whether the user has the right to verify or revoke an attribute. Before it was checking whether the attribute was present in the eservice, now it checks whether the attribute is present in the last descriptor.